src/ChipInput.js

```diff
@@ -324,6 +325,9 @@ class ChipInput extends React.Component {
   }
 
   handleAddChip (chip) {
+    if (this.props.onBeforeRequestAdd && !this.props.onBeforeRequestAdd(chip)) {
+      return this.setState({ preventChipCreation })
```

Comment:
A simple `return` should be sufficient here. Also, where does the `preventChipCreation` variable come from here?